controllers/admin/AdminLoginController.php

```diff
@@ -91,7 +91,7 @@ public function initContent()
                 $url = 'https://' . Tools::safeOutput(Tools::getServerName()) . Tools::safeOutput($_SERVER['REQUEST_URI']);
                 $warningSslMessage = $this->trans(
                     'SSL is activated. Please connect using the following link to [1]log in to secure mode (https://)[/1]',
-                    ['[1]' => '<a href="' . $url . '">', '[/1]' => '</a>'],
+                    ['_raw' => true, '[1]' => '<a href="' . $url . '">', '[/1]' => '</a>'],
```

Comment:
If I understand correctly, you are adding a deprecated parameters, so this will break in the next version

Reply:
This will not break but will be not necessary. On the other hand you will have to escape parameters before send them to the translation method. And we will have to communicate to prevent this behaviour to avoid security flaws.